students/ShujinCao/lesson09/assignment/database.py

```diff
@@ -0,0 +1,78 @@
+from pymongo import MongoClient
+import pandas as pd
+
+
+class MongoDBConnection():
+    """MongoDB Connection"""
+
+    def __init__(self, host='127.0.0.1', port=27017):
+        self.host = host
+        self.port = port
+        self.connection = None
+
+    def __enter__(self):
+        self.connection = MongoClient(self.host, self.port)
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.connection.close()
+
```

Comment:
Not sure exactly what is different in this file. I was thinking something useful to add to the MongoDBConnection context manager would be getting `mongo.connection.media` in there, since it is always run after the context manager is opened.